cannot build on windows

<!-- Please don't delete this template or we'll close your issue -->
<!-- Before creating an issue please make sure you are using the latest version of VuePress. -->

<!-- Please confirm you will submit an issue. -->
<!-- Issues which contain questions or support requests will be closed. -->
<!-- (Update "[ ]" to "[x]" to check a box) -->

- [x] I confirm that this is an issue rather than a question.

<!-- Please ask questions via following several ways. -->
<!-- https://vue-land.js.org/ -->
<!-- https://forum.vuejs.org/ -->
<!-- https://stackoverflow.com/questions/ask?tags=vuepress -->

## Bug report

#### Steps to reproduce

<!-- Please choose one of the following and and fill in the information. -->
<!-- (Update "[ ]" to "[x]" to check a box) -->

<!-- If you are reporting a bug that can ONLY be reproduced on your repository, PLEASE provide this repo link. That takes guessing work out of the way and saves us time. -->
<!-- If your repo isn't public, you can use `codesandbox` or `yarn create vuepress` to create a minimal reproduction -->

- [ ] I prepared a reproduction repo, here is the reproduction repo: 

- [x] This issue doesn't need a reproduction repro, here is the steps to reproduce

#### What is expected?

The command `yarn build` succeeds.

#### What is actually happening?

The script `build:components` fails with code 1.

```
>yarn build
yarn run v1.19.1
$ concurrently "rimraf lib" "npm:build:client" "npm:build:node" "npm:build:components"
[0] rimraf lib exited with code 0
[build:client]
[build:client] > @vuepress/plugin-blog@1.5.1 build:client .\vuepress-plugin-blog
[build:client] > tsc -p tsconfig.client.json
[build:client]
[build:components]
[build:components] > @vuepress/plugin-blog@1.5.1 build:components .\vuepress-plugin-blog
[build:components] > mkdir -p lib/client/components && cp -r src/client/components lib/client
[build:components]
[build:node]
[build:node] > @vuepress/plugin-blog@1.5.1 build:node .\vuepress-plugin-blog
[build:node] > tsc -p tsconfig.node.json
[build:node]
[build:components] The syntax of the command is incorrect.
[build:components] npm ERR! code ELIFECYCLE
[build:components] npm ERR! errno 1
[build:components] npm ERR! @vuepress/plugin-blog@1.5.1 build:components: `mkdir -p lib/client/components && cp -r src/client/components lib/client`
[build:components] npm ERR! Exit status 1
[build:components] npm ERR!
[build:components] npm ERR! Failed at the @vuepress/plugin-blog@1.5.1 build:components script.
[build:components] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[build:components]
[build:components] npm ERR! A complete log of this run can be found in:
[build:components] npm ERR!     cache\_logs\2019-11-27T02_53_41_037Z-debug.log
[build:components] npm run build:components exited with code 1
[build:client] npm run build:client exited with code 0
[build:node] npm run build:node exited with code 0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

#### Other relevant information

The cause of this issue is that `mkdir` on windows' cmd.exe does not support `-p` option and path separator `/` (allows only `\`).

- `npx vuepress info`:

--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
     "dev:node": "tsc -p tsconfig.node.json --watch",
     "build:client": "tsc -p tsconfig.client.json",
     "build:node": "tsc -p tsconfig.node.json",
-    "build:components": "mkdir -p lib/client/components && cp -r src/client/components lib/client",
+    "build:components": "cpx \"src/client/components/**/*\" lib/client/components",
     "dev:docs": "vuepress dev docs --temp docs/.temp",
     "build:docs": "vuepress build docs --temp docs/.temp",
     "dev:example": "node examples/launch.js dev",
@@ -38,6 +38,7 @@
   "devDependencies": {
     "concurrently": "^4.1.0",
     "conventional-changelog-cli": "^2.0.1",
+    "cpx": "^1.5.0",
     "inquirer": "^6.3.1",
     "nodemon": "^1.18.7",
     "rimraf": "^3.0.0",

